### PR TITLE
adopt a dog pull request

### PR DESCRIPTION
**Pull request by victorthoang**
a pull request for my html for adopt a dog.

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<!--insert meta data here-->
+	<title>Adoptadog - Seattle's premiere dog adoption agency</title>
```
Comment (jimmynono) [inline blog.html:5]: very important to include the meta tag referenced in week one's slides.

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
+body{
+    margin:0 0 0 0;
```
Comment (jimmynono) [inline styles/styles.css:2]: you could shorthand this to `margin: 0;`

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    background-color:#2c9ab7;
+    color:#ffffff;
+    background-image: url(../images/header-image.jpg);
+    background-size: cover;
+    width:100%;
+    height:300px;
+}
+
+.headertext{
+    padding-top: 60px;
```
Comment (jimmynono) [inline styles/styles.css:17]: I prefer REM's for padding. Just do the math 60px / 16 = 3.75rem

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    padding-left: 60px;
+}
+
+.ulnav{
```
Comment (jimmynono) [inline styles/styles.css:21]: this is kind of an award class name. What I would do is target the nav and then the ul
```
nav ul {
  // code
}
```

File: styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+a{
+    color:#42C1EA;
+    text-decoration:none;
+}
+
+.left
+{
+    float:left;
```
Comment (jimmynono) [inline styles/styles.css]: No floats. Find another way to lay out please.

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+    float:right;
+    width: 35%;
+    height: auto;
+    padding: 20px;
+    text-align: justify;
+}
+
+blockquote{
+    color:#81E3FD;
+    font-size: x-large;
```
Comment (jimmynono) [inline styles/styles.css]: Try to be more specific. Maybe a value in rem's

File: styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+.pagefooter{
+    background-color:#3a3a3a;
+    text-align:center;
+    color: #ffffff;
+    /* background-image: url(../images/adoptadog-logo.svg); */
+    background-size: cover;
+    width: 100%;
+    clear: both;
```
Comment (jimmynono) [inline styles/styles.css:62]: If you get rid of floats you wont need this

File: styles/styles.css [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+    list-style-type: none;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    background-color: transparent;
+    font-weight: bold;
+}
+
+.linavfooter{
+    padding: 0 5px 0 5px;
```
Comment (jimmynono) [inline styles/styles.css:77]: you could shorthand this to `padding: 0 5px`

File: styles/styles.css [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+  .grid-item {
+    /* background-color: rgba(255, 255, 255, 0.8); */
+    /* border: 1px solid rgba(0, 0, 0, 0.8); */
+    padding: 1rem;
+    font-size: 1rem;
+    text-align: center;
+  }
+
+  .flex-item{
+    width: 20%;
```
Comment (jimmynono) [inline styles/styles.css]: try using flex basis on width's on flex items

https://css-tricks.com/snippets/css/a-guide-to-flexbox/

File: styles/styles.css [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+    min-width: 20rem;
```
Comment (jimmynono) [inline styles/styles.css]: nice...this is a good way of handling this

File: js/adoptadog.js [hunk 11]
```diff
@@ -0,0 +1,10 @@
+function dogInfo(name, breed, fee) {
+    alert(name + ' is a ' + breed + ' and costs $' + fee + ' to adopt!');
+    }
+
+    let feeTotal = 0;
```
Comment (tylervanbrocklin) [inline js/adoptadog.js]: great work here, make sure to correct some of the indentations though

File: styles/styles.css [hunk 12]
```diff
@@ -0,0 +1,462 @@
...
+    display: block;
+    font-size: 1rem;
+    margin: 4px 2px;
+    cursor: pointer;
+    margin-left: auto;
+    margin-right: auto;
+  } */
+}
+@media screen and (min-width: 1000px) {
+  .box{
```
Comment (tylervanbrocklin) [inline styles/styles.css:248]: nice work implementing media queries, and just using min-width. It is not necessary to include both min and max widths. You broke up your screen sizes nicely!

File: styles/styles.css [hunk 13]
```diff
@@ -0,0 +1,517 @@
...
+
+@media screen and (min-width: 768px) {
+  .box{
+    width: 45%;
+    border-style: groove;
+  }
+  article img{
+    min-width: 75%;
+  }
+  aside{
```
Comment (jimmynono) [inline styles/styles.css:236]: at this point you'd want to  display the blog. You're off by 1 pixel. :)

File: js/adoptadog.js [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+let feeTotal = 0;
+
+function adoptionTotal(fee) {
+  feeTotal = fee + feeTotal;
+  alert('Total adoption fees: ' + '$' + feeTotal);
+  }
+
+const obj = {}; //declaring empty object
```
Comment (jimmynono) [inline js/adoptadog.js:13]: This looks....familiar. 😄

File: js/adoptadog.js [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+})
+
+$('.box').mouseenter(function(){
+  console.log('mouse has entered')
+  $(this).css('box-shadow', '10px 10px 10px #888');
+}).mouseleave(function(){
+  console.log('mouse has left')
+  $(this).css('box-shadow', 'none');
+})
+
```
Comment (tylervanbrocklin) [inline js/adoptadog.js:62]: Nice work with the hover effects!

File: checkout.html [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+            <a href="./dogs.html">Dogs</a>
+        </li>
+        <li class="linav">
+            <a href="./blog.html">Blog</a>
+        </li>
+    </ul>
+
+	<main>
+        <h1>Checkout</h1>
+        <div>
```
Comment (tylervanbrocklin) [inline checkout.html:34]: Very small, but I'd recommend seeing if you can use media queries to scale the size of your form. On desktop your fields take up the entire screen. On mobile this is great, but on desktop is a bit stretched out

**Pull request closed without merging**